### PR TITLE
[Element 2.0] Update documentation for Element 2.0

### PR DESCRIPTION
**Pull request by honglax**

### CODE REVIEW

File: packages/docs/docusaurus.config.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ module.exports = {
 		announcementBar: {
 			id: 'supportus',
 			content:
-				'⭐️ If you like Flood Element, give it a star on <a target="_blank" rel="noopener noreferrer" href="https://github.com/flood-io/element">GitHub</a>! ⭐️',
+				'🚀&nbsp;&nbsp;&nbsp;Flood Element v2.0 is now available, checkout the cool features <a target="_blank" rel="noopener noreferrer" href="https://www.flood.io/blog/whats-new-in-element-v2.0">here</a>&nbsp;&nbsp;🚀',
```
Comment (terryytran) [inline packages/docs/docusaurus.config.js:15]: There are 3 `&nbsp` at the beginning but only 2 at the end. Could we make it balance?
Also, please help me change the link to `https://www.flood.io/blog/element-2.0` (I'll update in Jira accordingly). Thanks!

File: packages/cli/README.md [hunk 2]
```diff
@@ -105,8 +104,8 @@ Setting a higher `--loop-count` could be useful for things like testing test dat
 
 Set the version of chrome to use. By default, `element run` uses the version of Chromium bundled with Puppeteer.
 
-- `--chrome` with no arguments will be equivalent to `'puppeteer'` (i.e. use the version of Chromium bundled with Puppeteer). Change it to `'stable'` to use the Chrome version installed on your system.
-- `--chrome /path/to/chrome` will use Chrome at the given path.
+-   `--chrome` with no arguments will be equivalent to `'puppeteer'` (i.e. use the version of Chromium bundled with Puppeteer). Change it to `'stable'` to use the Chrome version installed on your system.
+-   `--chrome /path/to/chrome` will use Chrome at the given path.
```
Comment (HieuLeKMS) [inline packages/cli/README.md:108]: This should be removed as `--chrome` is no longer a run argument

Comment (honglax) [reply packages/cli/README.md]: Resolved

File: packages/docs/versioned_docs/version-1.3/guides/editor.mdx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+title: Setting up youor
```
Comment (terryytran) [inline packages/docs/versioned_docs/version-1.3/guides/editor.mdx:2]: typo?

File: packages/docs/versioned_docs/version-2.0/guides/TestSettings.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+    
+### blockedDomains 
+`string\[]`  (Optional)   Blocks requests to a list a domains. Accepts partial matches using `*` or any matcher accepted by [Micromatch](https://github.com/micromatch/micromatch).
+    
+Matching is applied to the `hostname` only, unless the blocked domain contains a `:` in which case it will match against `hostname` and `port`.  
+
+Example:  `["*.google-analytics.com", "*:1337"]`  
+    
+### chromeVersion
+`puppeteer` or `stable` (Optional)   Specifies a version of Google Chrome to run the test
```
Comment (terryytran) [inline packages/docs/versioned_docs/version-2.0/guides/TestSettings.md:35]: Is this still valid and applicable?

File: packages/docs/versioned_docs/version-2.0/start/changelog.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
+---
+id: changelog
+title: Changelog
+---
+
+## 1.4.0
+Released: Nov 2020
```
Comment (terryytran) [inline packages/docs/versioned_docs/version-2.0/start/changelog.md:7]: need update to 1.5 and add 2.0?

**Review by terryytran — APPROVED** [review]
I've left some comments/questions. Please help resolve.

File: packages/docs/versioned_docs/version-1.3/guides/editor.mdx (hunk 3 above)
Comment (honglax) [reply packages/docs/versioned_docs/version-1.3/guides/editor.mdx:2]: Resolved

File: packages/docs/versioned_docs/version-2.0/guides/TestSettings.md (hunk 4 above)
Comment (honglax) [reply packages/docs/versioned_docs/version-2.0/guides/TestSettings.md:35]: My bad. Resolved.

File: packages/docs/versioned_docs/version-2.0/start/changelog.md (hunk 5 above)
Comment (honglax) [reply packages/docs/versioned_docs/version-2.0/start/changelog.md:7]: Just updated released date of 1.5. The released date of 2.0 will be defined later.

**Pull request merged**